core/src/test/java/io/confluent/rest/GzipHandlerIntegrationTest.java

```diff
@@ -0,0 +1,91 @@
+/*
+ * Copyright 2016 Confluent Inc.
```

Comment:
lol, doesn't really matter but i think your header template is out of date

Reply:
Ha, the Gzip filter was added in 2016 so we can say it’s keeping with the spirit of the code :)